dired.sublime-settings

```diff
@@ -33,6 +33,13 @@
   //     "git_path": ""
   "git_path": "git",
 
+  // Path to hg executable.
+  // Wildcards are supported, e.g. if you use TortoiseHg it would be:
+  //     "hg_path": "%ProgramFiles%/TortoiseHg/hg.exe"
```

Comment:
so… should i change those 2 lines or put `C:\Program Files\TortoiseHg\hg.exe` in the readme?